clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerResourceDefinitionRegistrar.java

```diff
@@ -269,7 +254,7 @@ public void registerSchema(SerializationContext context) {
                 // * The 2LC stack already overrides the interceptor for distribution caches
                 // * This renders Infinispan default 2LC configuration unusable as it results in a default media type of application/unknown for keys and values
                 // See ISPN-12252 for details
-                builder.addModule(PrivateGlobalConfigurationBuilder.class).serverMode(!ServiceLoader.load(ModuleCommandExtensions.class, loader).iterator().hasNext());
+//                builder.addModule(PrivateGlobalConfigurationBuilder.class).serverMode(!ServiceLoader.load(ModuleCommandExtensions.class, loader).iterator().hasNext());
```

Comment:
@pferraro Can you elaborate what's the situation with this? Isn't server mode already disabled by default, looking at `org.infinispan.configuration.internal.PrivateGlobalConfiguration#SERVER_MODE`? Can we just drop this block altogether?